lib/active_model/serializer.rb

```diff
@@ -69,12 +70,11 @@ def self.digest_caller_file(caller_line)
       serializer.class_attribute :_cache_digest # @api private : Generated
     end
 
-    # Serializers inherit _attributes and _attributes_keys.
+    # Serializers inherit serialized_attributes, _attributes_keys, and _reflections.
```

Comment:
typo: `s/serialized_attributes/_attributes_mappings`

typo: need to remove `_attributes_keys`